shipyard_blueprints/airtable/shipyard_airtable/cli/authtest.py

```diff
@@ -1,6 +1,6 @@
 import os
 import sys
-from shipyard_blueprints import AirtableClient
+from shipyard_airtable import AirtableClient
```

Comment:
Question across the board - in the authtests, will these import statements work given that some of these are not published in PyPi yet? I'm thinking yes since you navigate to the the proper working directory in the lambda script, but just checking.